packages/sdk-core/src/BacktraceCoreClient.ts

```diff
@@ -79,18 +79,23 @@ export abstract class BacktraceCoreClient {
     }
 
     /**
-     * Report database used by the client
+     * Client cached attachments
      */
-    public get database(): BacktraceDatabase | undefined {
-        return this._database;
+    public get attachments(): BacktraceAttachment[] {
```

Comment:
Make the return type `readonly BacktraceAttachment[]`.

Reply:
Changed to readonly